src/components/card/Description.vue

```diff
@@ -238,6 +253,65 @@ export default {
 				await this.saveDescription()
 			}, 2500)
 		},
+		async addLastAttachmmentOnCursor() {
+			const attachementLength = this.$store.getters.attachmentsByCard(this.card.id).length
+			if (attachementLength === 0) return
+
+			const attachment = this.$store.getters.attachmentsByCard(this.card.id)[attachementLength - 1]
+			const descStringLines = this.$refs.markdownEditor.easymde.value().split('\n')
+			const cursor = this.lastPasteImageCursor === null ? this.$refs.markdownEditor.easymde.codemirror.getCursor() : this.lastPasteImageCursor
+			let embed = ''
+			if ((attachment.type === 'file' && attachment.extendedData.hasPreview) || attachment.extendedData.mimetype.includes('image')) {
+				embed = '!'
+			}
+
+			const attachmentString = embed + '[📎 ' + attachment.data + '](' + this.attachmentPreview(attachment) + ')'
```

Comment:
This feels a bit like some code duplication with https://github.com/nextcloud/deck/pull/3951/files#diff-13efa8e07ea02fc9250f60af86b21f7ffb530e682b2ed240fa3ae5529794af7fR205, so maybe you could unify that or move some parts out to a helper function